src/client.h

```diff
@@ -422,6 +422,11 @@ class Client : public con::PeerHandler, public InventoryManager, public IGameDef
 		return m_csm_noderange_limit;
 	}
 
+	inline std::unordered_map<u32, u32> *GetHUDMapping()
```

Comment:
Should start with lowercase. Perhaps name it `getHUDTranslationMap` or `getHUDMap`?